packages/gatsby/src/utils/__tests__/__snapshots__/asset-path-registry.js.snap

```diff
@@ -0,0 +1,95 @@
+  "component---src-templates-template-docs-remote-packages-js.fd4497ec960200ce8103.css",
+  "component---src-templates-template-docs-remote-packages-js-68b7685d85bd50607d49.js",
+  "component---src-templates-template-docs-remote-packages-js-68b7685d85bd50607d49.js.map",
+  "component---src-pages-404-js-c31dd87a2028cf26b810.js",
+  "component---src-pages-404-js-c31dd87a2028cf26b810.js.map",
+  "component---src-pages-blog-tags-js-46843cd191f6d9a81483.js",
+  "component---src-pages-blog-tags-js-46843cd191f6d9a81483.js.map",
+  "5-5f0403c3cf66cc31c4fb.js",
+  "5-5f0403c3cf66cc31c4fb.js.map",
+  "component---src-pages-community-agencies-js-442a248e84f48ec18e11.js",
+  "component---src-pages-community-agencies-js-442a248e84f48ec18e11.js.map",
+  "component---src-pages-community-companies-js-4a9bb59d522d16bc0cd5.js",
+  "component---src-pages-community-companies-js-4a9bb59d522d16bc0cd5.js.map",
+  "component---src-pages-community-index-js-3c9cc70d04f424db3ffa.js",
+  "component---src-pages-community-index-js-3c9cc70d04f424db3ffa.js.map",
+  "component---src-pages-community-people-js-26220872f1564524ef20.js",
+  "component---src-pages-community-people-js-26220872f1564524ef20.js.map",
+  "component---src-pages-docs-actions-js-af5b12753a00718765fd.js",
+  "component---src-pages-docs-actions-js-af5b12753a00718765fd.js.map",
+  "component---src-pages-docs-browser-apis-js-f9529c1875808d214471.js",
+  "component---src-pages-docs-browser-apis-js-f9529c1875808d214471.js.map",
+  "component---src-pages-docs-index-js-c7323417d5798f1bcd94.js",
+  "component---src-pages-docs-index-js-c7323417d5798f1bcd94.js.map",
+  "component---src-pages-docs-node-apis-js-18a7ea5c808fce0d9402.js",
+  "component---src-pages-docs-node-apis-js-18a7ea5c808fce0d9402.js.map",
+  "component---src-pages-docs-ssr-apis-js-c4b332ac75f8093bdf42.js",
+  "component---src-pages-docs-ssr-apis-js-c4b332ac75f8093bdf42.js.map",
+  "component---src-pages-features-js-4a4a3f05f2ed8a3afa03.js",
+  "component---src-pages-features-js-4a4a3f05f2ed8a3afa03.js.map",
+  "component---src-pages-newsletter-js-b823f6af0f63d49f5bef.js",
+  "component---src-pages-newsletter-js-b823f6af0f63d49f5bef.js.map",
+  "component---src-pages-plugins-js.fd4497ec960200ce8103.css",
+  "component---src-pages-plugins-js-259ea07a4bf263d315c7.js",
+  "component---src-pages-plugins-js-259ea07a4bf263d315c7.js.map",
+  "42-d5c0588bf92953044654.js",
+  "42-d5c0588bf92953044654.js.map",
+  "component---src-pages-showcase-js-710ac9cbe8f04ccc5629.js",
+  "component---src-pages-showcase-js-710ac9cbe8f04ccc5629.js.map",
+  "39-72be356c49b30af6ec24.js",
+  "39-72be356c49b30af6ec24.js.map",
+  "component---src-pages-starters-js-5cea20767c8766bdcda4.js",
+  "component---src-pages-starters-js-5cea20767c8766bdcda4.js.map",
+  "component---src-pages-ecosystem-js-633f4f4caa8ef50afcbe.js",
+  "component---src-pages-ecosystem-js-633f4f4caa8ef50afcbe.js.map",
+  "component---src-pages-index-js-0a2a452fc816778a6329.js",
```

Comment:
This is currently _only_ the assets webpack knows about. It will be _all_ assets registered with `withAssetPrefix` once plugins implement this API.